common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -839,6 +839,7 @@ def submit_allowance_form(self, allowed_minutes, username):
         self.q(css='input#user_info').fill(username)
         self.q(css="input#addNewAllowance").click()
         self.wait_for_element_absence("div.modal div.modal-header", "Popup should be hidden")
+        self.wait_for_ajax()
```

Comment:
I wonder if it makes sense to move this up directly after the click? Is it possible for the element to disappear before the ajax call completes?

Also, please add a comment line on why we are waiting for Ajax. We're trying to avoid using that too much...folks that are less familiar with the front-end might be too aggressive with adding this wait statement, so we should be able to explain what it's doing when we do decide to put it in.